gvsbuild/projects/opus.py

```diff
@@ -13,31 +13,28 @@
 #  You should have received a copy of the GNU General Public License
 #  along with this program; if not, see <http://www.gnu.org/licenses/>.
 
-from gvsbuild.utils.base_builders import Meson
+from gvsbuild.utils.base_builders import CmakeProject
 from gvsbuild.utils.base_expanders import Tarball
 from gvsbuild.utils.base_project import project_add
 
 
 @project_add
-class Opus(Tarball, Meson):
+class Opus(Tarball, CmakeProject):
     def __init__(self):
-        Meson.__init__(
+        CmakeProject.__init__(
             self,
             "opus",
-            version="1.4",
+            version="1.5.1",
             repository="https://github.com/xiph/opus",
-            archive_url="https://github.com/xiph/opus/releases/download/v{version}/opus-{version}.tar.gz",
-            hash="c9b32b4253be5ae63d1ff16eea06b94b5f0f2951b7a02aceef58e3a3ce49c51f",
+            archive_url="https://downloads.xiph.org/releases/opus/opus-{version}.tar.gz",
+            hash="b84610959b8d417b611aa12a22565e0a3732097c6389d19098d844543e340f85",
             dependencies=[
                 "ninja",
                 "meson",
                 "pkgconf",
             ],
         )
 
-        self.add_param("-Dtests=disabled")
```

Comment:
can we disable tests and docs with cmake?

Reply:
Hi @pbor, thanks for the feedback. I updated the PR to disable the tests, I don't see an option to disable docs though.